assets/js/spotify-api.js

```diff
@@ -0,0 +1,21 @@
+function redirect(url, queryParams) {
+  if (queryParams) {
+    let queryString = "?";
+    for (const key in queryParams) {
+      queryString += `${key}=${queryParams[key]}&`
+    }
+    url += queryString;
+  }
+  location.href = url;
+}
+
+function generatePlaylistCover(e) {
+  const CLIENT_ID = "326af36f3fbd4a0dbcac034ee7216a2d";
+  
+  e.preventDefault();
+  redirect(`https://accounts.spotify.com/authorize`, queryParams={
```

Comment:
I think you only need these backtick quotes if you're gonna have a var in the string